event.py

```diff
@@ -0,0 +1,91 @@
+from typing import Iterable
+from copy import deepcopy
+from flask_restful import abort
+
+from timemodel import Time
+from eventdao import EventDAO
+
+
+DAYS = set(('Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'))
+
+class Event:
+    dao = EventDAO()
+
+    def __init__(self, name: str, days: Iterable[str], from_time: Time, to_time: Time, description=''):
+        assert set(days).issubset(DAYS)
+        assert from_time.in_seconds() <= to_time.in_seconds()
```

Comment:
The model should not be handling these assertions, the application will do all data validations before creating the Model class